erpnext/shopping_cart/cart.py

```diff
@@ -428,6 +428,9 @@ def get_party(user=None):
 		return customer
 
 def get_debtors_account(cart_settings):
+	if not cart_settings.get("payment_gateway_account"):
+		cart_settings.payment_gateway_account = get_default_payment_gateway_account(cart_settings)
```

Comment:
This field doesn't exist anymore, right? Keep it in a separate variable and take that ahead